### ISSUE TITLE
Formatting of Logs using Windows Notepad

### ISSUE THREAD

**Issue by supremestarhawk** [lisamelton/video_transcoding#287]
There may be a super simple solution to this that I'm not seeing, so my apologies if so. 

Since updating to a more recent version, when using Notepad on Windows, all logs are missing formatting making them difficult to read (Essentially all line breaks are missing). Opening the log in virtually any other text editor shows the correct formatting intact.

Is there anything I can do so log files retain formatting when opened in Notepad? The older version of video_transcoding I was using was before #213, so I'm wondering if that change made the difference?

**Comment by donmelton** [comment]
@cameronks The change in #213 actually makes it easier to open `.log` files in text editors. More than likely the problem is Notepad.exe itself. I recommend you download and install the free Notepad++ application and use that instead. See:

https://notepad-plus-plus.org/

I hope that helps.

**Comment by kvanh** [comment]
they're formatted with either Unix or Mac line endings. Until very recently the built in windows notepad didn't support these. Notepad++ that Don mentions is my go to editor on Windows and handles this automatically. Not quite BBEdit but superb for Windows.

**Comment by donmelton** [comment]
@kvanh Yes, if @cameronks is opening Mac- or Linux-generated `.log` files on Windows then the built-in Notepad.exe app will more than likely fail handling the line endings.

**Comment by cameronks** [comment]
Thank you both. I've had Notepad++ installed but not needed to use it much, but I will do so for logs now. 

Notepad was just super simplistic and all I needed to quickly view the logs from before.
